robot_smach_states/src/robot_smach_states/world_model.py

```diff
@@ -203,12 +204,16 @@ def __init__(self, robot, entityDes, objectIDsDes=None, searchArea="on_top_of",
                                                                                  entityDes),
                                        transitions={'unreachable': 'failed',
                                                     'goal_not_defined': 'failed',
-                                                    'arrived': 'SEGMENT'})
+                                                    'arrived': 'RISE'})
```

Comment:
There's no mechanism that ensures that the entity designator always resolves to the same entity (was already the case in this state). IMHO we should start thinking about this so that we *either* assert that is a locking designator and lock it (not so convenient for the user) *or* use userdata to this end (easier for the end-user, less reusable).

Reply:
Did you address this?

Reply:
I did not address this. Does this still fall in the scope of this PR? I think that locking the designator is the safest option however I fear this will break many applications.